specification/security/resource-manager/readme.md

```diff
@@ -1,3 +1,335 @@
+# Security Center

```

Comment:
This file contains unresolved merge conflicts, you may need to recheck this

Reply:
Resolved.
Can someone please merge?

Thanks